src/Impact.php

```diff
@@ -1548,6 +1577,7 @@ public static function prepareImpactNetwork(CommonDBTM $item): void
         self::printShowOngoingDialog();
         self::printEditCompoundDialog();
         self::printEditEdgeDialog();
+        echo Html::script("public/lib/cytoscape-popper.js");
```

Comment:
```suggestion
```

This file doesn't exist. The library is being bundled in the `public/lib/cytoscape.js` script.